bears/documentation/DocumentationStyleBear.py

```diff
@@ -81,10 +82,13 @@ def process_documentation(self,
                 # annotation directly.
                 if stripped_desc[0] != '':
                     stripped_desc.insert(0, '')
-
-            # Indent with 4 spaces.
-            stripped_desc = ('' if line == '' else ' ' * indent_size
-                             + line for line in stripped_desc)
+            # Indent with 4 spaces or a tab.
+            if use_spaces:
+                stripped_desc = ('' if line == '' else ' ' * indent_size +
+                                 line for line in stripped_desc)
+            else:
+                stripped_desc = ('' if line == '' else '\t' +
+                                 line for line in stripped_desc)
 
```

Comment:
you have to write a proper branch coverage test. like for this case you have to write 2 tests. first will cover the if part execution later will cover the else part.